From 2014e416c4a33c3c9f827d093879ba71b01600ac Mon Sep 17 00:00:00 2001
From: Chris Yeh <chris_yeh@phoenix.com>
Date: Tue, 22 Mar 2022 07:35:36 +0800
Subject: [PATCH] Add video reset patches to fix probe fail in video driver

Patches from SDK V08.00:
a242c92e4512 clk: aspeed: Add mapping for video reset
7e85007bf08e dt-bindings: clk: aspeed: Add reset for video
---
 drivers/clk/clk-aspeed.c                  | 1 +
 include/dt-bindings/clock/aspeed-clock.h  | 1 +
 include/dt-bindings/clock/ast2600-clock.h | 1 +
 3 files changed, 3 insertions(+)

diff --git a/drivers/clk/clk-aspeed.c b/drivers/clk/clk-aspeed.c
index 5e1c87bc8a99..3001dbe28180 100644
--- a/drivers/clk/clk-aspeed.c
+++ b/drivers/clk/clk-aspeed.c
@@ -286,6 +286,7 @@ static const u8 aspeed_resets[] = {
 	[ASPEED_RESET_PECI]	= 10,
 	[ASPEED_RESET_I2C]	=  2,
 	[ASPEED_RESET_AHB]	=  1,
+	[ASPEED_RESET_VIDEO]    =  6,
 
 	/*
 	 * SCUD4 resets start at an offset to separate them from
diff --git a/include/dt-bindings/clock/aspeed-clock.h b/include/dt-bindings/clock/aspeed-clock.h
index 41d531dd0b48..e9ae4549019b 100644
--- a/include/dt-bindings/clock/aspeed-clock.h
+++ b/include/dt-bindings/clock/aspeed-clock.h
@@ -54,5 +54,6 @@
 #define ASPEED_RESET_I2C		7
 #define ASPEED_RESET_AHB		8
 #define ASPEED_RESET_CRT1		9
+#define ASPEED_RESET_VIDEO		21
 
 #endif
diff --git a/include/dt-bindings/clock/ast2600-clock.h b/include/dt-bindings/clock/ast2600-clock.h
index 165bec96ebe7..1fabf75ea034 100644
--- a/include/dt-bindings/clock/ast2600-clock.h
+++ b/include/dt-bindings/clock/ast2600-clock.h
@@ -123,6 +123,7 @@
 #define ASPEED_RESET_PCIE_DEV_OEN	20
 #define ASPEED_RESET_PCIE_RC_O		19
 #define ASPEED_RESET_PCIE_RC_OEN	18
+#define ASPEED_RESET_VIDEO              6
 #define ASPEED_RESET_PCI_DP		5
 #define ASPEED_RESET_AHB		1
 #define ASPEED_RESET_SDRAM		0
-- 
2.17.1

